LaTeX_guidelines.md

```diff
@@ -101,5 +100,27 @@ Example:
 \end{definition}
 ```
 
+8. Write "i.e. ..... e.g. ..", NOT "i.e., ..... e.g., ..". I personally think comma is very unnecessary for these abbreviations.
+
+9. Use comma appropriately! A common mistake is to *forget* using comma before "FANBOYS" which are:
+- for
+- and
+- nor
+- but
+- or
+- yet
+- so
+
+Note that "FANBOYS" are used to connect *clauses*. 
+
+10. Do NOT use comma before "and" for the case of connecting words.
```

Comment:
can you add an 11. here

11. Use full stops appropriately with commas. If the quote can be replaced conceptually by emphasis/bold text, then the full stop should be outside the quotation marks. If the sentence ends within the quote the fullstop should be inside the quote
- e.g. 1: According to Byron, "Our burgers are fantastic."
- e.g. 2: The LaTeX guidelines are "fun".

Reply:
done